src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2421,12 +2421,12 @@ private function registerNotifierConfiguration(array $config, ContainerBuilder $
 
         if (ContainerBuilder::willBeAvailable('symfony/fake-chat-notifier', FakeChatTransportFactory::class, ['symfony/mailer'])) {
```

Comment:
The factory definition may have been removed earlier if any of `$parentPackages` is missing
```suggestion
        if (ContainerBuilder::willBeAvailable('symfony/fake-chat-notifier', FakeChatTransportFactory::class, array_merge($parentPackages, ['symfony/mailer']))) {
```

Reply:
Already done before your comment :-)